Remove mssql filter from testthat.

chrisknoll opened the pull request:
This is to remove the filter from the test_check to only look at mssql tests.  This will verify if the TravisCI will execute without timeout across all dbms.

chrisknoll commented:
Ok, we're back to a timeout after 10 minutes after enabling the oracle and mssql testing.  I do not have the knowledge of what's happening behind the scenes with travis, or why testthat does not report any output until all the tests are complete (thus causing the ci build to time out), but in order for this to get resolved, someone with more access to the environment and expertise with he build system needs to get involved.

chrisknoll commented:
@schuemie , @msuchard , any thoughts you can shed on this would be appreciated.

chrisknoll commented:
@schuemie , I'm trying to add a travis_wait call when running tests, however, this commit removed the script: block in .travis.yml so I'm actually not sure how tests are being executed now: 50fb6e1e5ed6ee47dc55b14b37f4320a8704cbc2.  I think we need to extend the timeout for when the tests are being called, but I don't know how to do that.

t-abdul-basser commented:
With that commit it looks like we moved from older R-supporting Travis approach to the recommended travis-R. The problem is that the style of the former is imperative while the latter is declarative. See (here)[https://github.com/travis-ci/travis-ci/issues/5469] for a possible work around.

chrisknoll commented:
Ok, using the information found here: https://github.com/travis-ci/travis-ci/issues/3849, I've added a script block to invoke the build and check, but the check is having a wait specified of 30 mins.  I've also added an after_failure block to spit out the contents of the *.fail files.  This should allow tests to complete but it's finding the error in oracle and postgresql (which is good, probably realated to the casting). So if I can get the tests to complete, I'd like to have this PR merged into master, and then we'll make another PR to remove the CAST() from analysis 109 that is casting a DATE to a DATE (and hopefully this is the only place where we're seeing this error)

chrisknoll commented:
Ok, tests completed, and it's identified the problem in the postgresql, but also in the oracle tests, around the same lines of code, but for a different reason I believe: when creating the temp_dates temp table:

```sql
Error:
execute JDBC update query failed in dbSendUpdate (ORA-00909: invalid number of arguments
)

SQL:
CREATE TABLE 
  OHDSI.xp5wfdi2temp_dates
  AS
SELECT
   DISTINCT 
  EXTRACT(YEAR FROM observation_period_start_date) AS obs_year,
  TO_DATE(CONCAT(TO_CHAR(EXTRACT(YEAR FROM observation_period_start_date) ), '01', '01') , 'yyyymmdd') AS obs_year_start,
  TO_DATE(CONCAT(TO_CHAR(EXTRACT(YEAR FROM observation_period_start_date) ), '12', '31') , 'yyyymmdd') AS obs_year_end
FROM
  CDMV5.observation_period
```
Now, i'm not exactly sure what the invalid number of arguments is here, EXTRACT for the obs_year looks correct, the CONCAT looks correct, and the TO_DATE can be called with 2 params (a char, an then a format). 

I can file the oracle error separately and address in a followup PR that addresses the postgres and oracle errors.  But This PR will need to be merged in first, so @t-abdul-basser , if you could merge this into master, I'd appreciate it.

Note: you can use the squelch-and-merge option to reduce all these commits into a single commit. Thanks!

t-abdul-basser commented:
Understood. Reviewed and done.
